tapir/statistics/templatetags/statistics.py

```diff
@@ -20,6 +22,16 @@ def purchase_statistics_card(context, tapir_user):
     return context
 
 
+@register.inclusion_tag("statistics/tags/credit_account_card.html", takes_context=True)
+def credit_account_card(context, tapir_user):
+    user_credits = CreditAccount.objects.filter(tapir_user=tapir_user)
+    context["last_credits"] = user_credits.order_by("-credit_date")[:10]
+    context["actual_credit"] = "{:.2f}".format(
+        user_credits.aggregate(total_credit=Sum("credit_amount"))["total_credit"]
+    )
```

Comment:
This throws an error when there is no CreditAccount object for the given member.